frontend/lib/shopify-storefront-sdk/generate-schema.js

```diff
@@ -14,30 +14,13 @@ if (shopName == null || shopName.length === 0) {
    throw new Error('CODEGEN_STOREFRONT_SHOP_NAME is not set');
 }
 
-(async () => {
-   try {
-      await spawn(
-         `gq https://${shopName}.myshopify.com/api/${version}/graphql.json --introspect -H 'X-Shopify-Storefront-Access-Token: ${apiKey}' -H 'Content-Type: application/json' > ${__dirname}/schema.graphql`
-      );
-   } catch (error) {
-      console.log('ERROR:', error);
+child.exec(
+   `gq https://${shopName}.myshopify.com/api/${version}/graphql.json --introspect -H 'X-Shopify-Storefront-Access-Token: ${apiKey}' -H 'Content-Type: application/json' > ${__dirname}/schema.graphql`,
+   {},
+   (error) => {
+      if (error) {
+         console.error('ERROR:', error);
+         process.quit(1);
+      }
```

Comment:
What's the reasoning behind this refactor?

Reply:
I was trying to get it to run and I was getting a failure with no output. I was trying to get to the root of why it was failing so I tried launching with a different node function thinking `spawn()` may not have been capturing the output correctly. In the end, I realized the output was being redirected at the end of the command and the failure message was in the generated file and my change had no affect. I left the refactor cause the resulting code was simpler.

That said, it doesn't change the result, so I'm ok dropping this change.